src/zoo/Animal.java

```diff
@@ -16,4 +20,11 @@ public Animal(String name, int age, boolean isWild) {
     public String giveVoice(){
         return "usual Animal voice ";
     }
+    public String doVildThings(boolean isWild){
```

Comment:
I think the name should be `wild` ;) not `vild` :)

Reply:
true:), done